src/shared/destinations.ts

```diff
@@ -114,6 +114,24 @@ const destinationConfigs: DestinationConfig[] = [
       return `${repoUrl}/blob/HEAD/CHANGELOG.md`;
     },
   },
+  {
+    keywords: ["d"],
+    generateUrl: async (packageName) => {
```

Comment:
Can we deal with breaking change somehow here for the cli 🤔  e.g. pseudocode

```
if (from > cli@1.0.0) forward('h')
```

Reply:
Nice idea, I like! Happy to review it in a separate PR once we get this one in 👍 I’ll release cli@v2, v1 will end up broken and then we’ll fix the server to cater for the differences. I’m pretty sure that `d` is not used that widely, so a temp behaviour mismatch for v1 won’t be the end of the world 🙂